index.html

```diff
@@ -0,0 +1,138 @@
+          <div class="reasons-one__background-img"></div>
+        </div>
+      </section>
+
+      <section class="to-try">
+        <div class="to-try__inner">
+          <ul class="to-try__list">
+            <li class="to-try__item">
+              <img src="" class="to-try__item--house-pic">
+              <h2 class="to-try__header">Accomodation</h2>
+              <p class="to-try__paragraph to-try__paragraph__list">We recommend to live in a real motel, everything is like in a movie!</p>
+            </li>
+            <li class="to-try__item">
+              <img src="" class="to-try__item--gift-pic">
+              <h2 class="to-try__header">Souvenirs</h2>
+              <p class="to-try__paragraph to-try__paragraph__list">Not only Chinese, but also the local products!</p>
+            </li>
+            <li class="to-try__item">
+              <img src="" class="to-try__item--burger-pic">
+              <h2 class="to-try__header">Food</h2>
+              <p class="to-try__paragraph to-try__paragraph__list">Always order a Brand Burger, you will not be disappointed!</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="reasons"><!-- THERE IS THE BRIDGE OF THE DEVIL - No. 2 - Yes, you can go through it! If... -->
+        <div class="reasons__inner reasons__inner--blue">
+          <h2 class="reasons__header">There's a Devil's Bridge</h2>
+          <p class="reasons__paragraph">— №2 —</p>
+          <p class="reasons__paragraph">Yes, you can go through it! If you dare of course.</p>
+        </div>
+        <div class="reasons__background-img"></div>
+      </section>
+
+      <section class="reasons"><!-- SMALL AREA - No. 3 - ALL SITES are located -->
+        <div class="reasons__inner">
+          <h2 class="reasons__header">Sall Area</h2>
+          <p class="reasons__paragraph">— №3 —</p>
+          <p class="reasons__paragraph">All the attractions are very close</p>
+        </div>
+      </section>
+
+      <section class="reasons"><!-- BEAUTIFUL ROAD - No. 4 - TO GO TO SEDONU FROM LAS VEGAS completely... -->
+        <div class="reasons__inner">
+          <h2 class="reasons__header">Beautiful Road</h2>
+          <p class="reasons__paragraph">— №4 —</p>
+          <p class="reasons__paragraph">Go to Sedona Las Vegas it's anything but not boring!</p>
+        </div>
+      </section>
+
+      <section class="reasons"><!-- LITTLE TOURISTS - No. 5 - Most travel to Grand Canyon and crowd... -->
+        <div class="reasons__inner">
+          <h2 class="reasons__header">Small Tourism</h2>
+          <p class="reasons__paragraph">— №5 —</p>
+          <p class="reasons__paragraph">Most of the turrist travel to the Grand Canyon and it is over crowded. In here is oposite!</p>
+        </div>
+      </section>
+
+      <section class="search"><!-- INTERESTED? Indicate the expected dates of your trip and we are -->
```

Comment:
Search name is misleading and will confuse develpers expecting to see search. I would use something more like` book-hotel` or `find-hotel`